instagram/src/components/Authentication/Authenticate.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import LoginPage from '../Login/Login';
+
+const Authenticate = App => Login => {
```

Comment:
Looks good. Only thing I would change is that you typically want HOC's to start with a lowercase letter by convention. :)
```suggestion
const authenticateUser = App => Login => {
```